### PR TITLE
MSI-2161: UI Components in Store Pickup admin UI.

### PR DESCRIPTION
**Pull request by nmalevanec**
### Fixed Issues (if relevant)

1. https://github.com/magento/inventory/issues/2161: UI Components in Store Pickup admin UI

### Manual testing scenarios (*)

1. Create order with 'store pickup' delivery method.
2. Verify "Notify Order is Ready for Pickup" button is present on admin order page.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: InventoryInStorePickupSalesAdminUi/view/adminhtml/layout/sales_order_view.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="content">
-            <block class="Magento\InventoryInStorePickupSalesAdminUi\Block\Adminhtml\Order\View\ReadyForPickup" name="sales_order_ready_for_pickup"/>
```
Comment (sidolov) [inline InventoryInStorePickupSalesAdminUi/view/adminhtml/layout/sales_order_view.xml:11]: Is `Magento\InventoryInStorePickupSalesAdminUi\Block\Adminhtml\Order\View\ReadyForPickup` should be removed after such changes or it's used somewhere else?

**Review by sidolov — CHANGES_REQUESTED** [review]

File: InventoryInStorePickupSalesAdminUi/view/adminhtml/layout/sales_order_view.xml (hunk 1 above)
Comment (nmalevanec) [reply InventoryInStorePickupSalesAdminUi/view/adminhtml/layout/sales_order_view.xml:11]: @sidolov, yes, looks like I've forgotten to remove it. Done.

**Comment by sidolov** [comment]
@magento run Functional Tests B2B

**Comment by nmalevanec** [comment]
Wrong solution.

**Pull request closed without merging**